### PR TITLE
[14.0] account_avatax_exemption_base: change license to LGPL

### PR DESCRIPTION
**Pull request by atchuthan**
cc: @SodexisTeam

### CODE REVIEW

**Review by dnplkndll — APPROVED** [review]
functionally works well on US exemption management

**Comment by odoo-sh** [comment]
Did QA and it is working well.

File: account_avatax_exemption_base/__manifest__.py [hunk 1]
```diff
@@ -7,16 +7,15 @@
     """,
     "website": "https://github.com/OCA/account-fiscal-rule",
     "author": "Sodexis, Odoo Community Association (OCA)",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
     "depends": [
-        "account_avatax",
-        "account_avatax_sale",
+        "mail",
+        "account",
```
Comment (dreispt) [inline account_avatax_exemption_base/__manifest__.py:13]: I'm quite sure this must depend on "account_avatax".
In that case, "mail" and "account" dependencies will not be explicitly needed.

**Review by dreispt — CHANGES_REQUESTED** [review]
I have a concern with module dependencies. Can you check?

File: account_avatax_exemption_base/__manifest__.py (hunk 1 above)
Comment (atchuthan) [reply account_avatax_exemption_base/__manifest__.py:13]: This module doesn't have direct relation to account_avatax.  Also, we are trying to make this module as LGPL license such that we can add further customization of enterprise module such as  sign and documents related to account_avatax_exemption in the future.

Comment (atchuthan) [reply account_avatax_exemption_base/__manifest__.py:13]: @dreispt please let us know if the above answer resolves your issue

**Comment by amkarthik** [comment]
Looks good to me

**Review by amkarthik — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

**Pull request merged**